docs/extend/modular-resources/examples/triton.md

```diff
@@ -0,0 +1,185 @@
+[Install `viam-server` on the board](/installation/prepare/jetson-agx-orin-setup/) and connect to the robot.
+
+## Configuration
+
+{{< tabs name="Add the Triton modular service">}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot’s page in the [Viam app](https://app.viam.com).
+Click on the **Services** subtab and click the **Create service** button.
+Search for `triton`, then select the version from the [Registry](https://app.viam.com/registry).
+Give your resource a name of your choice and click **Create**.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+Add the following to your `"modules"` array:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "type": "registry",
+  "name": "viam_mlmodelservice-triton-jetpack",
+  "module_id": "viam:mlmodelservice-triton-jetpack",
+  "version": "0.4.0"
+}
+```
+
+Add the following to your `"services"` array:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "name": "my-triton-module",
+  "type": "mlmodel",
+  "namespace": "rdk",
+  "model": "viam:mlmodelservice:triton"
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The model will now be configured with a card like the following:
+
+![The triton service card in the Viam app config builder, showing deployment options.](/extend/modular-resources/triton/triton-config-builder.png)
+
+Note that the parameters shown, `"model_path"`, `"label_path"`, and `"num_threads"` are not applicable for this module, and should be left blank.
+Ignore this card's interface and move to [creating a model repository](#create-a-repository-to-store-the-ml-model-to-deploy).
+
+### Create a repository to store the ML model to deploy
+
+Currently, you must manually create a Triton [model repository](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_repository.html).
+On your robot's Jetson computer, create a [structured repository](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_repository.html) under the `~/.viam` directory.
+The exact subpath under `~/.viam` does not matter.
+
+For example, after unpacking the module, to add the [EfficientDet-Lite4 Object Detection](https://tfhub.dev/tensorflow/efficientdet/lite4/detection/2) model, place the model repository under `~/.viam/triton/repository`:
+
+```sh { class="command-line" data-prompt="$"}
+$ tree ~/.viam
+~/.viam
+├── cached_cloud_config_05536cf6-f8a6-464f-b05c-bf1e57e9d1d9.json
+└── triton
+    └── repository
+        └── efficientdet-lite4-detection
+            ├── 1
+            │   └── model.savedmodel
+            │       ├── saved_model.pb
+            │       └── variables
+            │           ├── variables.data-00000-of-00001
+            │           └── variables.index
+            └── config.pbext
+```
+
+The <file>config.pbext</file> file must exist, but at least for TensorFlow models it can be empty.
```

Comment:
```suggestion
The <file>config.pbext</file> file must exist, but at least for TensorFlow models it can be empty.
For other types of models, please consult the [Triton Server model repository documentation](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_repository.html) for details on the correct contents of the <file>config.pbext</file> file.
```